src/lib/components/sml-versions/SMLVersionForm.svelte

```diff
@@ -30,6 +35,30 @@
     onSubmit: (data: SMLVersionData) => onSubmit(trimNonSchema(data, smlVersionSchema))
   });
 
+  const add = () => {
+    $data.arch = $data.arch.concat({ id: '', SMLVersionLinkID: '', platform: '', link: '' });
+  };
+
+  const remove = (i) => () => {
+    $data.arch = $data.arch.filter((_, index) => index != i);
+  };
+
+  function isAdd(index) {
+    if (index != $data.arch.length - 1) {
+      return true;
+    } else {
+      return false;
+    }
+  }
+
+  function isDel() {
+    if ($data.arch.length == 1) {
+      return true;
+    } else {
+      return false;
+    }
```

Comment:
Please change the function to just `$data.arch.length == 1`

Reply:
https://github.com/satisfactorymodding/smr-frontend/pull/47/commits/9c0893e0353c97b347f9f0152ec590c614d255ac

I removed it and isAdd, but it still functions as needed.